### PR TITLE
rectify differences with main

### PR DESCRIPTION
**Pull request by kylomite**
##feature
- update mainpage structure
- update css to responsibility
- implement functionality for edge cases with buttons

### CODE REVIEW

File: src/MainPage/MainPage.css [hunk 1]
```diff
@@ -8,124 +8,85 @@
...
-  background-image: url('../Utilities/Images/Box_Forest_BDSP.png');
-}
-
-.play-container .play-area-8 {
-  background-image: url('../Utilities/Images/Box_River_BDSP.png');
-}
-
-.play-container .play-area-9 {
-  background-image: url('../Utilities/Images/Box_Savanna_BDSP.png');
+.HUD {
```
Comment (jchirch) [inline src/MainPage/MainPage.css:38]: Glad we got this component where we want it on the main page, I believe al lot of the HUD work was done by Karl, but is not reflected in this PR due to merge conflicts in deleted branch.

File: src/MainPage/MainPage.css [hunk 2]
```diff
@@ -8,124 +8,85 @@
...
+  width: 35%;
+  margin: 1em;
+  color: #000000;
+  outline: 5px ridge;
```
Comment (jchirch) [inline src/MainPage/MainPage.css:42]: potential change in the future would be to change styling unit from px to something more dynamic. Save for suture refactor.

**Comment by MDelarosa1993** [comment]
Looks great! The structure and styling choices create a clean, functional layout while maintaining a fun and interactive user experience. Well done!

File: src/MainPage/MainPage.js [hunk 3]
```diff
@@ -43,9 +43,20 @@ function MainPage() {
...
-  const [playAnim, setPlayAnim] = useState(0);
-  const levelUp = () => {
+  let bgArray = [beachImg, caveImg, checkImg, cityImg, cragImg, desertImg, forestImg, savannahImg, seafloorImg, skyImg, snowImg, volcanoImg, theBeach]
+  let bgTemp = bgArray[Math.round(Math.random() * bgArray.length)];
+  
+  useEffect(() => {
+    fetchData(2);
+  }, [])
+
+  useEffect(() => {
```
Comment (jchirch) [inline src/MainPage/MainPage.js:55]: A much needed refactor, looks like the background will change upon loading, not with every use state change, which was not conducive for those sensitive to strobing lights.

File: src/MainPage/MainPage.js [hunk 4]
```diff
@@ -148,106 +154,81 @@ function MainPage() {
...
-  // }
   function playWithCurrentPokemon() {
+  
+    let pokemonSprite = document.getElementById("currentRender")
 
-    console.log(playAnim)
-    setPlayAnim(1);
-
-    console.log(playAnim)
+    pokemonSprite.classList.remove('jump')
```
Comment (jchirch) [inline src/MainPage/MainPage.js:169]: They said it couldnt be done! Very smart work around for our animation problems using a toggle and timeout, I will use this in future projects.

File: src/MainPage/MainPage.js [hunk 5]
```diff
@@ -148,106 +154,81 @@ function MainPage() {
...
-              <Button variant="secondary" onClick={handleHelpVisible}>
-                Close
-              </Button>
-            </Modal.Footer>
-          </Modal>
-        </button>
-      </header>
-
-      <div className='play-container'>
+      <div className='play-container' style={{ backgroundImage: `url(${background})` }}>
```
Comment (jchirch) [inline src/MainPage/MainPage.js:202]: Looks much cleaner. Greatly cleaning up the div soup aids in accessibility with more semantic HTML and I imagine makes the CSS display easier to work with.

**Review by jchirch — APPROVED** [review]

**Pull request merged**